docs/source/dynamo/get-started.rst

```diff
@@ -6,13 +6,12 @@ significant speedups the newer your GPU is.
 
 .. code:: python
 
-   from torch._dynamo import optimize
    import torch
    def fn(x, y):
        a = torch.cos(x).cuda()
        b = torch.sin(y).cuda()
        return a + b
-   new_fn = optimize("inductor")(fn)
+   new_fn = torch.compile(fn, backend="inductor")
```

Comment:
The default backend is inductor, not sure if you want to be explicit here or not

Reply:
Yes it's deliberate, I did want to showcase how it's easy to use different compilers. I might remove them in another iteration maybe as soon as this week lol.

In all cases will add you to another PR explaining compile configs better